Blur example with copying pointers instead of the content of the array

rjanalik opened the pull request:
Implemented only the function _nocopies
copy(out) could be avoided, but it would be complicated (copyout + update device)

This should be in the solution, not here

rjanalik committed:
Blur example with copying pointers instead of the content of the array

Implemented only the function _nocopies
This should be in the solution, not here
copy(out) could be avoided, but it would be complicated

vkarak reviewed (changes requested):
Hi @rjanalik, thanks for the suggestion. Can you patch this in the solutions? And leave here the TODOS?

rjanalik commented:
Yes, I will do it over the Christmas period.
Or do you need it ASAP?

rjanalik committed:
Merge branch 'master' of github.com:vkarak/openacc-training

rjanalik committed:
Update blur nocopies

Update blur_twice_gpu_nocopies() such that it copies pointers
instead of the content of the ocntent of the array.

vkarak reviewed (approved):
Thanks @rjanalik for your suggestion!
